### PR TITLE
Onward video series

### PR DESCRIPTION
**Pull request by jamesgorrie**
## What does this change?

We're looking to see if people want to see more series.
We know this wasn't the case for certain content, especially articles, but we think it might be for video.
## What is the value of this and can you measure success?

We will be measuring whether we have more people use the related content component in it's series-ified version. @harrysalmon
## Does this affect other platforms - Amp, Apps, etc?

No
## Screenshots

more-from-desktop
more-from-mini
## Request for comment

@akash1810

### CODE REVIEW

**Comment by akash1810** [comment]
Can't comment inline as on mobile, but..

Do we need to `showFields(all)`? Could we just ask for what we're rendering?

**Comment by jamesgorrie** [comment]
> Do we need to showFields(all)
I'll refine the query once we've done the test, it doesn't add much overhead.

File: onward/app/controllers/MostViewedVideoController.scala [hunk 1]
```diff
@@ -1,12 +1,39 @@
...
+    val edition = Edition(request)
+    getResponse(ContentApiClient.search(edition)
+      .tag(series)
+      .showTags("all")
+      .showFields("all")
+    ).map { response =>
+      val videos = response.results.toList.map(apiContent => {
+        val content = Content.make(apiContent)
+        Video.make(content)
+      }).take(6)
```
Comment (akash1810) [inline onward/app/controllers/MostViewedVideoController.scala]: could we use capi's page-size param?

File: dev-build/conf/routes [hunk 2]
```diff
@@ -114,6 +114,7 @@ GET            /tagged.json
 GET            /:mediaType/section/:sectionId/*seriesId.json                                                                     controllers.MediaInSectionController.renderSectionMediaWithSeries(mediaType, sectionId, seriesId)
 GET            /:mediaType/section/:sectionId.json                                                                               controllers.MediaInSectionController.renderSectionMedia(mediaType, sectionId)
 GET            /video/most-viewed.json                                                                                           controllers.MostViewedVideoController.renderMostViewed()
+GET            /video/in-series/*series.json                                                                                     controllers.MostViewedVideoController.renderInSeries(series: String)
```
Comment (akash1810) [inline dev-build/conf/routes]: do the other `routes` need this path added to too?

Comment (jamesgorrie) [inline dev-build/conf/routes]: Good point - done

**Comment by akash1810** [comment]
👍

File: onward/app/controllers/MostViewedVideoController.scala (hunk 1 above)
Comment (jamesgorrie) [inline onward/app/controllers/MostViewedVideoController.scala]: Yip - good ol Copy / Paste + nothinking

**Pull request merged**